### ISSUE TITLE
Add a "spec maturity summary" field

### ISSUE THREAD

**Issue by jyasskin** [GoogleChrome/chromium-dashboard#3113]
**Is your feature request related to a problem? Please describe.**

I recently added a [requirement](https://www.chromium.org/blink/launching-features/#new-feature-prepare-to-ship:~:text=If%20your%20specification%20isn%27t%20a%20modification%20of%20an%20existing%20specification%2C%20include%20a%20one%2Dline%20spec%20maturity%20summary%20from%20someone%20outside%20your%20team%20(like%20your%20spec%20mentor)%20who%20has%20done%20a%20review.) to the blink launch process to 

> If your specification isn't a modification of an existing specification, include a one-line [spec maturity summary](https://www.chromium.org/blink/spec-mentors/#reviewing-the-specification) from someone outside your team (like your [spec mentor](https://www.chromium.org/blink/spec-mentors)) who has done a review.

Chrome Status should support that requirement.

**Describe the solution you'd like**

Ideally, this would be a field that only the assigned spec mentor(s) could set, but even an editor-writable field would be fine. This should be a textarea and not just a single-line input, in case mentors want to write more than the minimum requirement, as [several](https://groups.google.com/a/chromium.org/g/blink-dev/c/JTPl7fM64Lc/m/OfBZRdjEAAAJ) [mentors](https://groups.google.com/a/chromium.org/g/blink-dev/c/tpw8wW0VenQ/m/Tw0nZULfDQAJ) [have been doing](https://groups.google.com/a/chromium.org/g/blink-dev/c/igFixT5n7Bs/m/mfq1kM3kDQAJ) [ad-hoc](https://groups.google.com/a/chromium.org/g/blink-dev/c/dZ0NRwh7cvs/m/LiQq-ZHZDQAJ).

**Comment by jrobbins** [comment]
So, this new field in prepare-to-ship would be in addition to the existing "Standard maturity" enum in the prototyping stage, right?
![image](https://github.com/GoogleChrome/chromium-dashboard/assets/17365/0b39393f-6ddb-4eee-87db-fd763aaae955)

**Comment by jyasskin** [comment]
Yes. This is "specification maturity", about the rigor of the specification document itself. The "Standard maturity" field is about how far it's progressed through the standards process. (Often, a higher 'standard maturity' implies better rigor in the documents, but not always, and we have several high-rigor documents in community groups.)
